src/attached_probe.cpp

```diff
@@ -657,4 +664,31 @@ void AttachedProbe::attach_hardware()
   }
 }
 
+void AttachedProbe::attach_watchpoint(int pid)
+{
+  // We require the pid to make the initial implementation simpler.
+  // Technically we could walk /proc/*/exe and find which pids symlink to
+  // the provided executable path. This would enable multiple watchpoint
+  // probes in a single bpftrace program.
```

Comment:
What about new processes?

Reply:
Yes, that'd be a problem. Let's keep it to 1 pid for now.